src/libraries/System.ComponentModel.Primitives/src/ILLink/ILLink.LinkAttributes.xml

```diff
@@ -0,0 +1,55 @@
+<linker>
```

Comment:
Some of these are applied in other libraries in dotnet/runtime.  Is this change going to cause them to be trimmed out as part of each assembly's build?

Reply:
No, we don't trim any attributes as part of build optimizations/trimming. They will be trimmed when you publish output with trimming enabled